src/extension/app/constants.js

```diff
@@ -59,4 +57,16 @@ export const EXTERNAL_EVENTS = {
 export const MODALS = {
   WAIT: 'wait',
   ERROR: 'error',
+  DELETE: 'delete',
+};
+
+/**
+ * Modal Events
+ * @enum {string}
+ */
+export const MODAL_EVENTS = {
+  CLOSED: 'closed',
+  CANCELLED: 'cancelled',
```

Comment:
I would use present tense for all of them
```suggestion
  CLOSE: 'close',
  CANCEL: 'cancel',
```